ocaml/idl/datamodel_cluster.ml

```diff
@@ -219,6 +219,9 @@ let t =
        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:Int "live_hosts"
            ~default_value:(Some (VInt 0L))
            "Current number of live hosts, according to the cluster stack"
+       ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:Int "expected_hosts"
+           ~default_value:(Some (VInt 0L))
+           "Total number of hosts expected by the cluster stack"
```

Comment:
I'm not sure I understand the exact meaning of "expected by the cluster stack". Is that the amount of nodes that the cluster stack has been configured with?

Reply:
While I understand that it's useful to expose the information for debugging, the difference in semantics of these numbers is quite subtle.

Reply:
> I'm not sure I understand the exact meaning of "expected by the cluster stack". Is that the amount of nodes that the cluster stack has been configured with?

Precisely